docs/source/api_ref_utilities.rst

```diff
@@ -4,6 +4,19 @@ torchtune.utils
 
 .. currentmodule:: torchtune.utils
 
+
+.. _checkpointing_label:
+
+Checkpointing
+-------------
+
+.. autosummary::
+    :toctree: generated/
+    :nosignatures:
+
+    FullModelHFCheckpointer
+    FullModelMetaCheckpointer
```

Comment:
The tutorial discusses 3 checkpointers, but the torchtune one is left out here. Is the thinking that users don't really have to use the torchtune one? If so, then shall we just remove it from tutorial, as we're presenting user with info that they're not really going to use?